src/python/setup.py

```diff
@@ -0,0 +1,7 @@
+#!/bin/env python3
```

Comment:
Same here.

Reply:
Actually, this file should not be there, removed.